luatest/server_v2.lua

```diff
@@ -1,6 +1,6 @@
 --- Class to manage Tarantool instances, version 2.
 --
--- @classmod luatest.server2
+-- @classmod luatest.server_v2
```

Comment:
Otherwise, luatest/init.lua:31 should be changed

```diff
  -- @see luatest.server_v2
- luatest.ServerV2 = require('luatest.server_v2')
+ luatest.ServerV2 = require('luatest.server2')
```

Reply:
Fixed.